### ISSUE TITLE
Multiple errors on update to Scribble 9.3.2, cause unclear

### ISSUE THREAD

**Issue by JustAGuyAFAIK** [JujuAdams/Scribble#580]
Hey guys. Just recently updated to the latest build, but it seems the entirety of Scribble is broken, even the test project. Various errors include:
`
___________________________________________
############################################################################################
ERROR in action number 1
of  Step Event0 for object obj_test_markdown:
Variable <unknown_object>.__bbox_array(100107, -2147483648) not set before reading it.
 at gml_Script_region_detect@anon@18846@__scribble_class_element@__scribble_class_element (line 654) -             var _bbox_array = _region.__bbox_array;
############################################################################################
gml_Script_region_detect@anon@18846@__scribble_class_element@__scribble_class_element (line 654)
gml_Object_obj_test_markdown_Step_0 (line 1) - var _region = element.region_detect(room_width/2, room_height/2, mouse_x, mouse_y);
`

`--------------------------------------------------------------
An error has occured:ERROR in action number 1
of Draw Event for object obj_introstory:
unable to convert string "/wave" to number
 at gml_Script___process_event_stack@anon@13509@__scribble_class_typist@__scribble_class_typist (line 524) -                         var _duration = (array_length(_event_data) >= 1)? real(_event_data[0]) : SCRIBBLE_DEFAULT_DELAY_DURATION;

--------------------------------------------------------------`

`
___________________________________________
############################################################################################
ERROR in action number 1
of Draw Event for object obj_test_typewriter_preskip:
ScribbleDX:
Cannot call text element methods using the result from .draw()
This can occur in two situations:
  1. scribble().draw().method();
  2. t = scribble().draw(); t.method()
Instead use:
  1. scribble().method().draw();
  2. t = scribble(); t.method(); t.draw();
 
 at gml_Script___scribble_error (line 60) -     show_error("ScribbleDX:\n" + _string + "\n ", true);
############################################################################################
gml_Script___scribble_error (line 60)
gml_Script___error@anon@99@__scribble_class_null_element@__scribble_class_null_element (line 6) -         __scribble_error("Cannot call text element methods using the result from .draw()\nThis can occur in two situations:\n  1. scribble().draw().method();\n  2. t = scribble().draw(); t.method()\n\nInstead use:\n  1. scribble().method().draw();\n  2. t
gml_Script_flash@anon@881@__scribble_class_null_element@__scribble_class_null_element (line 45) -         __error();
gml_Object_obj_test_typewriter_preskip_Draw_0 (line 1) - scribble(test_string).draw(x, y, typist).flash(c_white, 1);
`

And many more. Seems like there wasn't enough testing done before releasing... Huge breaking changes.

**Comment by JujuAdams** [comment]
Last error is on you. Other ones I will look into.

**Comment by JujuAdams** [comment]
Unable to repro second error nor can I see a way that could occur by reading the code. First error is addressed by [this hotfix](https://github.com/JujuAdams/Scribble/releases/download/9.3.2/scribble_9_3_2_1.yymps).

I will need to know more details about these "many more" errors to help further.

**Comment by JustAGuyAFAIK** [comment]
The last error was coming from the test case project itself, not from me.
`scribble(test_string).draw(x, y, typist).flash(c_white, 1);`
![image](https://github.com/user-attachments/assets/7967ac05-6731-4e6f-9679-bbde4f855a28)

Fixed code: `scribble(test_string).flash(c_white, 1).draw(x, y, typist);`

The second error is caused by stacking a `[delay]` and `[wave]` together. I was able to reliably reproduce the error on the test project as well.

https://github.com/user-attachments/assets/edf3131c-ac08-40e3-82c5-1bd5c0237573
